WordPress-Theme/ruleset.xml

```diff
@@ -3,11 +3,9 @@
 	<!-- For more information: https://make.wordpress.org/themes/handbook/review/ -->
 	<description>Standards any Theme to be published on wordpress.org should comply with.</description>
 
-
 	<!-- No PHP short open tags allowed. -->
 	<!-- Covers: https://github.com/Otto42/theme-check/blob/master/checks/phpshort.php -->
 	<rule ref="Generic.PHP.DisallowShortOpenTag" />
-
-
+	<rule ref="WordPress-Theme" />
```

Comment:
Could you leave this out for the moment as it will make it easier to merge in the future as it may be added in another PR before hand.